packit/config/package_config_validator.py

```diff
@@ -67,17 +69,31 @@ def validate(self) -> str:
                 f"the file is present."
             )
 
-        if not schema_errors:
-            return f"{self.config_file_path.name} is valid and ready to be used"
+        synced_files_errors = []
+        if config:
+            synced_files_errors = [
+                f for f in iter_srcs(config.synced_files) if not Path(f).is_file()
```

Comment:
Ouch, it seems the wording in our doc [is somewhat misleading](https://packit.dev/docs/configuration/#synced_files), although the examples are right: these paths can be also directories, not only files. They should be checked with `.exists()`.